frontend/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ProjectListView from "./components/ProjectListView/ProjectListView";
+import "./App.css";
+
+const App: React.FC = () => {
```

Comment:
Import `FC` in the line 1 and use FC directly instead of using React.FC.

```suggestion
const App: React.FC = () => {
```
```suggestion
const App: FC = () => {
```